Documentation/Packaging/MRTK_Packages.md

```diff
@@ -2,16 +2,48 @@
 
 The Mixed Reality Toolkit (MRTK) is a collection of packages that enable cross platform Mixed Reality application development by providing support for Mixed Reality hardware and platforms.
 
-The MRTK ships via the following Unity packages:
+MRTK is available as [asset](#asset-packages) (.unitypackage) packages and via the [Unity Package Manager](#unity-package-manager).
+
+
+## Asset packages
+
+The MRTK asset (.unitypackage) can be downloaded from [GitHub](https://github.com/microsoft/MixedRealityToolkit-Unity/releases). 
+
+Some of the benefits of using asset packages include:
+
+- Available for Unity 2018.4 and newer
+- Can configure which files are imported
```

Comment:
Nit: It feels a bit misleading on this one - you definitely can omit things but it's notoriously hard for anyone who isn't us to do this. 

It can be hard to say it's a benefit in this case.